### PR TITLE
Add test coverage for id as string

### PR DESCRIPTION
**Pull request by cesine**
When we upgraded from 3.5.3 to 3.6.2 we found that the `id` was no longer a string if an integer was passed.

https://jsonapi.org/format/#document-resource-object-identification

> The values of the `id` and `type` members MUST be `strings`.

Is this intended, for now I included a test case to show you what we are looking at:

- [x] Add a test case to demo the issue
- [ ] Fix the issue

### CODE REVIEW

File: test/serializer.js [hunk 1]
```diff
@@ -897,7 +912,7 @@ describe('JSON API Serializer', function () {
         firstName: 'Lawrence',
         lastName: 'Bennett',
         address: {
-          id: '54735697e16624ba1eee36bf',
+          id: 123,
```
Comment (SeyZ) [inline test/serializer.js]: Why did you change this test here?

File: test/serializer.js [hunk 2]
```diff
@@ -934,6 +949,9 @@ describe('JSON API Serializer', function () {
         type: 'addresses'
       });
 
+      expect(json.included[1]).to.have.property('id')
+        .equal('123');
+
```
Comment (SeyZ) [inline test/serializer.js]: Same question here

**Comment by SeyZ** [comment]
@cesine thanks for your contribution. Your commit with the fix has been merged through this PR: https://github.com/SeyZ/jsonapi-serializer/pull/190

File: test/serializer.js (hunk 1 above)
Comment (cesine) [reply test/serializer.js]: Coverage for strings in an array.

File: test/serializer.js (hunk 2 above)
Comment (cesine) [reply test/serializer.js]: Added expectations that id in an array will be a string as well.

**Comment by cesine** [comment]
@SeyZ thanks for finding and merging the fix!
